yt_dlp/extractor/fox.py

```diff
@@ -20,7 +20,7 @@
 
 
 class FOXIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?fox\.com/watch/(?P<id>[\da-fA-F]+)'
+    _VALID_URL = r'https?://(?:www\.)?fox(?:sports)?\.com/(?:watch|replay)/(?P<id>[\da-fA-F]+)'
```

Comment:
could also do it this way, though I don't think it matters
```suggestion
    _VALID_URL = r'https?://(?:www\.)?fox(?:\.com/watch|sports\.com/replay)/(?P<id>[\da-fA-F]+)'
```

Reply:
I'd rather keep it this way in case a foxsports.com/watch or fox.com/replay appears